packages/@aws-cdk/aws-cloudfront-origins/README.md

```diff
@@ -76,3 +76,20 @@ new cloudfront.Distribution(this, 'myDist', {
 ```
 
 See the documentation of `@aws-cdk/aws-cloudfront` for more information.
+
+## Failover Origins
+
```

Comment:
I'd start this section with a brief (stolen) documentation section explaining what failover origins are. 

Something like this paragraph (perhaps slightly tweaked):
```suggestion
You can set up CloudFront with origin failover for scenarios that require high availability. To get started, you create an origin group with two origins: a primary and a secondary. If the primary origin is unavailable, or returns specific HTTP response status codes that indicate a failure, CloudFront automatically switches to the secondary origin.

```
*Source: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/high_availability_origin_failover.html*

Reply:
Done!